Makefile

```diff
@@ -1013,7 +1013,8 @@ endif
 # Packaging components
 #
 ##############################
-PACKAGE_TARGETS = package_installer package_all_compress
+PACKAGE_TARGETS = package_installer package_ground package_matlab
```

Comment:
Do we need all these?  I think it's unlikely we'll ship matlab separately, for instance...